storage/reads/aggregate_resultset.go

```diff
@@ -23,6 +23,23 @@ type windowAggregateResultSet struct {
 	err          error
 }
 
+func IsAscendingWindowAggregate(req *datatypes.ReadWindowAggregateRequest) bool {
+	if len(req.Aggregate) != 1 {
+		return false
+	}
+
+	if req.Aggregate[0].Type == datatypes.AggregateTypeLast {
+		if req.Window == nil {
```

Comment:
Can you add a comment for this section while you're here explaining the checks in this code? It wasn't here previously so no requirement but it would be nice to comment this.